api/envoy/extensions/clusters/dynamic_forward_proxy/v3/cluster.proto

```diff
@@ -54,3 +75,26 @@ message ClusterConfig {
   //
   bool allow_coalesced_connections = 3;
 }
+
+// [#next-free-field: 6]
+message SubClustersConfig {
+  // The :ref:`service discovery type <arch_overview_service_discovery_types>`
+  // to use for resolving a sub cluster. Only STRICT_DNS and LOGICAL_DNS are supported here.
```

Comment:
What is the point of supported LOGICAL_DNS? Isn't this basically the same as what is already done? Should we just support STRICT_DNS for now?

Reply:
yep, only support `STRICT_DNS` for now does make sense.
it could be useful when we deprecate the current API.
will mark this field as `not-implemented-hide`.